src/Collection/common.js

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
missing `// @flow` at the top of the file (that should be an ESLint rule if you can find that really quick)

Reply:
I added 
```
 flowtype/require-valid-file-annotation:
    - 2
    - always
```

it didn't give an error on linting. But, when I save the file, it automatically insert insert flow annotation. I am not sure why it doesn't trigger error though.

Reply:
if you are happy with the rule, I can add it. I quickly checked the `eslint-plugin-flowtype` repo and there are few issues to go through. Sorry, I haven't used eslint for a while and don't want to spend too much time. thanks

Reply:
here is the link for that rule https://github.com/gajus/eslint-plugin-flowtype#eslint-plugin-flowtype-rules-require-valid-file-annotation

Reply:
Ok, nvm then. I’ll check this myself as I’d have to configure tests to be ignored. But please add @flow to this file

Reply:
ok done.